data/topic-schema/src/main/kotlin/net/corda/schema/Schemas.kt

```diff
@@ -34,24 +34,78 @@ class Schemas {
         fun getRPCResponseTopic(topic: String) = "$topic.resp"
     }
 
+    /**
+     * Config read topic schema
+     */
+    class Config {
+        companion object {
+            const val CONFIG_TOPIC = "config.topic"
```

Comment:
Is there anything that ensures that topic names listed here match those defined in the YAML? Better still, could these be generated from the YAML?

Reply:
I agree, that is a must.

The easiest may be for these to loaded from the yaml at runtime. We only have these const as we refer to them in the code, but the cost of loading the yaml file at start-up into a map is probably negligible, and would be easier than code-gen.

Reply:
I think the .kt file is probably required. I would love code generation, but it is a pain. We definitely could add a unit test though that reflectively scans the file and checks